README.md

```diff
@@ -84,14 +84,18 @@ A Kubernetes Secret also needs to be defined and mounted to the Pod. The secret
 {
   "AuthSecret": "base64 encoded auth secret",
   "S3SecretKey": "s3 secret key",
-  "S3AccessKey": "s3 access key"
+  "S3AccessKey": "s3 access key",
+  "GithubClientID": "github client ID",
+  "GithubClientSecret": "github client secret"
 }
 ```
 
 `AuthSecret` is the secret for authenticating requests. This is needed by the clients to communicate with kontinuous through JWT.
 
 `S3SecretKey` and `S3AccessKey` are the keys needed to access minio (or S3).
 
+`GithubClientID` and `GithubClientSecret` are optional and only required for Github only authentication. (See below)
```

Comment:
The docs don't seem quite clear. Is it only "Step 1" (registering the application on GH) that is needed and not the full oAuth dance or does a user also need to provide the JWT from Step 3 (as the AuthSecret)?